### PR TITLE
Mutant races now are selectable via the traits menu instead of spacebux

### PR DESCRIPTION
**Pull request by Superlagg**
## About the PR 

Lizards, cows, skeletons, and roaches are now chosen through character traits instead of SpaceBux. Also increases the max traits to 8 from 7.

## Why's this needed? 

Seems like a better idea than picking it for free from character creation.

## Changelog

```
(u)Superlagg:
(*)Lizards, cows, skeletons, and roaches are removed from SpaceBux and moved to character traits.
```

### CODE REVIEW

**Comment by Kyle2143** [comment]
why make them worth -1 and add an extra point to everyone in traits. I think either make it -2 or don't add an extra

**Comment by ZeWaka** [comment]
I like this change

File: code/datums/traits.dm [hunk 1]
```diff
@@ -1030,3 +1030,67 @@ obj/trait/pilot
 	id = "atheist"
 	points = 0
 	isPositive = 0
+
+/obj/trait/lizard
+	name = "Reptilian (-1) \[Species\]"
+	cleanName = "Reptilian"
+	icon_state = "lizardT"
+	desc = "Trade in your human status and become a scaly, hissing lizardperson!"
```
Comment (Kyle2143) [inline code/datums/traits.dm:1038]: Change this scaly thing.

File: code/datums/traits.dm [hunk 2]
```diff
@@ -1030,3 +1030,67 @@ obj/trait/pilot
...
+		if(ishuman(owner))
+			var/mob/living/carbon/human/H = owner
+			H.set_mutantrace(/datum/mutantrace/lizard)
+		return
+
+/obj/trait/cow
+	name = "Bovine (-1) \[Species\]"
+	cleanName = "Bovine"
+	icon_state = "cowT"
+	desc = "Trade in your human status and become a furry, mooing cow!"
```
Comment (Kyle2143) [inline code/datums/traits.dm:1054]: Change this furry reference too.

**Comment by Kyle2143** [comment]
Someone mentioned that, and the reason for those is that these are either jokes poking fun at furries/scalies, or implying that these are references to that content, or that it content made "for" furries/scalies to rp with (certainly they can rp as that, but it's sort of disingenuous).

**Comment by Sartorius7** [comment]
You're not a bad guy for saying so. I don't think adding furry stuff for its own sake is a thing I think Goon should be doing. We're not that kind of server.

**Comment by ZeWaka** [comment]
> You're not a bad guy for saying so. I don't think adding furry stuff for its own sake is a thing I think Goon should be doing. We're not that kind of server.

Imo, we shouldn't add furry content because it's furry content. We should be adding content that is cool, unique, has good gameplay mechanics, and fits with our server. 

Some stuff like the 'cow' race fits into this, because lol it's just a perfectly normal human, what's wrong with you... I also like the recent PRs to make lizards more of a gameplay shift instead of just being an aesthetics thing, etc.

File: code/datums/traits.dm [hunk 3]
```diff
@@ -1030,3 +1030,67 @@ obj/trait/pilot
...
+		if(ishuman(owner))
+			var/mob/living/carbon/human/H = owner
+			H.set_mutantrace(/datum/mutantrace/skeleton)
+		return
+
+/obj/trait/roach
+	name = "Roach (-1) \[Species\]"
+	cleanName = "Roach"
+	icon_state = "roachT"
+	desc = "Trade in your human status and become a chitinous, skittering humanoid cockroach!"
```
Comment (Tarmunora) [inline code/datums/traits.dm:1086]: not even a single nod to Kafka smh

Comment (Sartorius7) [reply code/datums/traits.dm:1086]: What would Gregor Samsa think?

File: code/datums/traits.dm (hunk 1 above)
Comment (Superlagg) [reply code/datums/traits.dm]: Expunged all reference to scales.

File: code/datums/traits.dm (hunk 2 above)
Comment (Superlagg) [reply code/datums/traits.dm]: Expunged all reference to cows. And fur.

File: code/datums/traits.dm (hunk 3 above)
Comment (Superlagg) [reply code/datums/traits.dm]: Bam, get Kafka'd on.

**Comment by Kyle2143** [comment]
Wow, those are all fantastic. The only improvement I can think of would be to somehow make a nod to the HUMAN's speech text replacer that replaces HUMAN with HUMAN.

**Review by Tarmunora — APPROVED** [review]

**Comment by Sovexe** [comment]
Unsure what will happen when people who have the persistent bank purchases that no longer exist attempt to play

**Review by ZeWaka — CHANGES_REQUESTED** [review]
> Unsure what will happen when people who have the persistent bank purchases that no longer exist attempt to play

runtime

**Comment by Superlagg** [comment]
> > Unsure what will happen when people who have the persistent bank purchases that no longer exist attempt to play
> 
> runtime

Okay I think I made it not do that anymore. Like, I added a check to make it see if the thing we bought is actually in the list of things, and delete the thing they have if they still have it when they spawn. Dunno how well it works, can't really do much without the thing that makes spacebux work properly, but it did let me purchase things (that I set to be free) and did give me the expected error when I manually called the proc that gives the thing with random objects.

File: code/modules/economy/persistent_bank.dm [hunk 4]
```diff
@@ -16,8 +16,14 @@
...
 
+		var/thing_we_bought
+		if(usr.client.persistent_bank_item && (usr.client.persistent_bank_item in persistent_bank_purchaseables))
+			thing_we_bought = usr.client.persistent_bank_item
+		else
+			thing_we_bought = "Nothing!"
+
 		var/ret = "<p style=\"font-size:125%;\">BALANCE :  <b>[usr.client.persistent_bank]</b></p><br/>"
-		ret += "<p style=\"font-size:110%;\">HELD ITEM :  <b>[usr.client.persistent_bank_item ? usr.client.persistent_bank_item : "none"]</b></p><br/>"
+		ret += "<p style=\"font-size:110%;\">HELD ITEM :  <b>[thing_we_bought]</b></p><br/>"
```
[the remark this answers is not in the document]
Comment (Superlagg) [reply code/modules/economy/persistent_bank.dm]: Done!

**Review by Sovexe — APPROVED** [review]

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**